tools/topology/topology1/platform/intel/intel-generic-dmic-rtnr.m4

```diff
@@ -1,135 +0,0 @@
-#
-# Topology for digital microphones array
-#
```

Comment:
Does this patch need squashed in the series as looks like we are adding this file earlier ?

Reply:
The commits are squashed. The indentation and commit message are also modified. Thanks!